PlatformAwareEnergy/README.md

```diff
@@ -0,0 +1,117 @@
+
+- Deploy the [custom metrics pipeline](https://github.com/intel/platform-aware-scheduling/blob/master/telemetry-aware-scheduling/docs/custom-metrics.md). Once the custom metrics is deployed correctly, running 
+
+  
+  ```bash
+  kubectl get --raw /apis/custom.metrics.k8s.io/v1beta1 | grep nodes | jq . 
+  ```
+  should return a number of metrics that are being collected by prometheus node exporter, scraped by prometheus and passed to the Kubernetes Metrics API by the Prometheus Adapter. One example metrics collection is as follows:
+  
+
+  ```bash
+  kubectl get --raw /apis/custom.metrics.k8s.io/v1beta1/nodes/*/memory_Cached_bytes | jq .
+
+  {
+    "kind": "MetricValueList",
+    "apiVersion": "custom.metrics.k8s.io/v1beta1",
+    "metadata": {
+      "selfLink": "/apis/custom.metrics.k8s.io/v1beta1/nodes/%2A/memory_Cached_bytes"
+    },
+    "items": [
+      {
+        "describedObject": {
+          "kind": "Node",
+          "name": "kind-worker",
+          "apiVersion": "/v1"
+       },
+        "metricName": "memory_Cached_bytes",
+        "timestamp": "2022-01-21T08:32:03Z",
+        "value": "7560781824",
+        "selector": null
+      },
+      {
+        "describedObject": {
+          "kind": "Node",
+          "name": "kind-worker2",
+          "apiVersion": "/v1"
+      },
+        "metricName": "memory_Cached_bytes",
+        "timestamp": "2022-01-21T08:32:03Z",
+        "value": "7560638464",
+        "selector": null
+      }
+    ] 
+  }
+  ```
+You can explore the custom metrics values on prometheus dashboard as well. Here is an example:
+
+![](images/prometheus_custom_metrics.png)
+
+In order to be certain the raw metrics are available look at a specific endpoint output from the above command e.g. kubectl get --raw /apis/custom.metrics.k8s.io/v1beta1/nodes/*/health_metric | jq .
+
+- Follow the steps described [here](https://github.com/intel/platform-aware-scheduling/tree/master/telemetry-aware-scheduling#extender-configuration). There's an extender configuration example [here](https://github.com/intel/platform-aware-scheduling/tree/master/telemetry-aware-scheduling/deploy/extender-configuration). 
+
+- Deploy TAS(Telemetry Aware Scheduling) following the steps [here](https://github.com/intel/platform-aware-scheduling/tree/master/telemetry-aware-scheduling#deploy-tas).
+
+- Deploy [Power Driven Scheduling and Scaling with CPU telemetry in Kubernetes](https://github.com/intel/platform-aware-scheduling/blob/master/telemetry-aware-scheduling/docs/power/README.md). Once you follow all the steps described above you will have the following:
+
+  - Deploy collectd and prometheus adapter. This will exposes node's energy metrics to kubernetes and prometheus. Check the following picture for an example. 
```

Comment:
```suggestion
  - Deploy collectd and prometheus adapter. This will expose node's energy metrics to kubernetes and prometheus. Check the following picture for an example. 
```